src/iconsManifest/supportedExtensions.ts

```diff
@@ -668,6 +668,11 @@ export const extensions: IFileCollection = {
       format: FileFormat.svg,
       disabled: true,
     },
+    {
+      icon: 'bak',
+      extensions: ['bak'],
```

Comment:
I think `.bak` is the most common, but I would be open to add the `.back` and `.backup` extensions as well.

Reply:
let's add them.